includes/classes/Jobber.php

```diff
@@ -22,7 +23,7 @@ class Jobber {
 	 *
 	 * @var string
 	 */
-	protected $api_url = 'https://jobber-prod.10upmanaged.io/jobber';
+	protected static $api_url = 'https://jobber-prod.10upmanaged.io';
```

Comment:
Would be great to provide a way to override this with a constant for easier testing locally or on staging